src/OrchardCore.Themes/TheAgencyTheme/Views/MenuItemLink-HtmlMenuItem.liquid

```diff
@@ -1,2 +1,2 @@
 {% assign link = Model.ContentItem.Content.HtmlMenuItemPart %}
-<a class="nav-link" href="{{ link.Url | href }}">{{ link.Html | raw }}</a>
\ No newline at end of file
+<a class="nav-link" href="{{ link.Url | href }}"{% if link.Target %} target="{{link.Target}}" {% endif %}>{{ link.Html | raw }}</a>
```

Comment:
New line at the end of the file and add target attribute by default with _self as default value